README.md

```diff
@@ -28,35 +27,25 @@ engines:
     enabled: true
     config:
       languages:
-        YOUR_LANGUAGE:
+      - ruby
+      - javascript
 ```
 
-This will tell Code Climate to run the duplication engine with `YOUR_LANGUAGE`.
-You can also specify a `paths` array under `YOUR_LANGUAGE` which use Ruby's
-[`Dir.glob`][glob] format.
-
-For example, all JavaScript and JSX files:
+This will tell the duplication engine to analyze ruby and javascript files.
```

Comment:
I think we consistently capitalize Ruby and JavaScript -- might want to audit the whole README for consistency.